app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      before_action :authenticate_user!
+
+      def create
+        rent = Rent.new(rent_params)
+        if rent.save!
+          Api::V1::RentsMailer.new_rent(rent.id).deliver_later
```

Comment:
Hum, this not should be in this PR? In the next one?

Reply:
The email notification PR was already accepted and merged to this branch